code/modules/clothing/suits/wintercoats.dm

```diff
@@ -227,8 +235,17 @@
 	desc = "A white winter coat with green markings. Warm, but wont fight off the common cold or any other disease. Might make people stand far away from you in the hallway. The zipper tab looks like an oversized bacteriophage."
 	icon_state = "coatviro"
 	inhand_icon_state = "coatviro"
+/obj/item/clothing/suit/hooded/wintercoat/medical/viro/Initialize(mapload)
+	. = ..()
+	allowed += list(
+		/obj/item/storage/bag/bio,
+
+	)
```

Comment:
```suggestion
	allowed += /obj/item/storage/bag/bio
```

Reply:
Will change and push them once I get connection back